waspc/test/UtilTest.hs

```diff
@@ -18,6 +18,17 @@ spec_camelToKebabCase = do
     camel ~> kebab = it (camel ++ " -> " ++ kebab) $ do
       camelToKebabCase camel `shouldBe` kebab
 
+spec_projectNameToAppIdentifier :: Spec
+spec_projectNameToAppIdentifier = do
+  "foobar" ~> "foobar"
+  "s3" ~> "s3"
+  "foo-bar-bar" ~> "fooBarBar"
+  "todo-App" ~> "todoApp"
+  "TODO-app" ~> "TODOApp"
+  where
+    kebab ~> camel = it (kebab ++ " -> " ++ camel) $ do
+      projectNameToAppIdentifier kebab `shouldBe` camel
```

Comment:
@sodic I implemented this as a way to make those tests shorter and easier to read!
You are right that there are common parts of it, but I am not sure how viable would be making it a higher order function -> because it also needs to accept the mapping function. If you have a good idea how to do it, I am open for it! But I think it also works relatively well as local trick. At least it is easy to read and understand.

Reply:
Oh, right, it's a binary operator and we'd need to pass a function. Never mind then!